packages/widget-chat/src/components/activity-item-post-actions/index.js

```diff
@@ -0,0 +1,26 @@
+import React, {PropTypes} from 'react';
+import classNames from 'classnames';
+
+import Icon from '../icon';
+
+import styles from './styles.css';
+
+export default function ActivityItemPostActions(props) {
+  function handleOnDelete() {
+    const {id, onDelete} = props;
+    onDelete(id);
+  }
+
+  return (
+    <div className={classNames(`post-actions`, styles.postActions)}>
+      <div className={classNames(`post-actions-item`, styles.postActionsItem)}>
+        <Icon handleOnClick={handleOnDelete} title="Delete this message" type="delete" />
```

Comment:
Should the icon really be handling the delete? I would think the container should be handling this, and the icon itself should be only presentation. It seems awkward to hand the delete handler down another level.